internal/pipe/cask/testdata/TestFullPipe/uninstall.rb.golden

```diff
@@ -0,0 +1,82 @@
+# This file was generated by GoReleaser. DO NOT EDIT.
+cask "uninstall" do
+  desc "Run pipe test formula and FOO=foo_is_bar"
+  homepage ""
+  version "1.0.1"
+
+  livecheck do
+    skip "Auto-generated on release."
+  end
+
+  binary "uninstall"
+
+  depends_on formula: "ash"
+  depends_on formula: "bash"
+  depends_on cask: "fish"
+  depends_on cask: "powershell"
+  depends_on formula: "zsh"
+
+  on_macos do
+    on_intel do
+      url "https://dummyhost/download/v1.0.1/bin.tgz"
+      sha256 "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+    end
+    on_arm do
+      url "https://dummyhost/download/v1.0.1/bin.txz"
+      sha256 "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+    end
+  end
+
+  on_linux do
+    on_intel do
+      url "https://dummyhost/download/v1.0.1/bin.tar.zst"
+      sha256 "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+    end
+  end
+
+  conflicts_with formula: "bash"
+  conflicts_with cask: "fish"
+
+  postflight do
+    system "echo"
+    touch "/tmp/hi"
+  end
+
+  caveats do
+    "don't do this uninstall"
+  end
+
+  service "foo.plist"
+
+  uninstall launchctl: [
+    "launchctl1",
+    "launchctl2",
+    "launchctl3",
+  ]
+
+  uninstall quit: [
+    "quit1",
+    "quit2",
+    "quit3",
+  ]
+
+  uninstall login_item: [
+    "loginitem1",
+    "loginitem2",
+    "loginitem3",
+  ]
+
+  uninstall trash: [
+    "trash1",
+    "trash2",
+    "trash3",
+  ]
+
+  uninstall delete: [
+    "delete1",
+    "delete2",
+    "delete3",
+  ]
```

Comment:
I'm not sure if this will work, AFAIK it needs to be like:
```suggestion
  uninstall launchctl: [
    "launchctl1",
    "launchctl2",
    "launchctl3",
  ],
  quit: [
    "quit1",
    "quit2",
    "quit3",
  ],
  login_item: [
    "loginitem1",
    "loginitem2",
    "loginitem3",
  ],
  trash: [
    "trash1",
    "trash2",
    "trash3",
  ],
  delete: [
    "delete1",
    "delete2",
    "delete3",
  ]
```

Reply:
I asked some other maintainers to weigh in. I'm not 100% sure.

Reply:
It is technically possible to repeat the uninstall stanza, but as far as I know we don't have any casks that actually do it, so we always ask to have the uninstall stanzas combined during PR review. 

In terms of the template here, it's unlikely that you would use `trash` for uninstall, and most `zap` stanzas only ever use `trash`.